tests/default.nix

```diff
@@ -38,4 +38,5 @@ pkgs.runCommand
       --subst-var-by prootTest "${bootstrap.customPkgs.prootTermuxTest}" \
       --subst-var-by installationDir "${bootstrap.config.build.installationDir}" \
       --subst-var-by homeDir "${bootstrap.config.user.home}" \
+      --subst-var-by flakeRootDir "${toString ../.}"
```

Comment:
Will that make fakedroid ignore uncommitted changes and always build environment from the commit it itself is built from?

Reply:
It will build with all changes tracked by git. Behaves the same as every other interaction with flakes in git repositories.